tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -37,6 +37,7 @@ def indexer_record(use_case_id: UseCaseKey, org_id: int, string: str) -> int:
 
 
 @region_silo_test
+@freeze_time()
```

Comment:
alternatively -- does it also fix it if time is only retrieved once?  I assume the flakiness is due to a second ticking over in the loop?

Reply:
do you mean if we move `now = time.time()` out of the all loops and rewrite it `started = (now // 60 - 3 + minute) * 60`?

Reply:
I think so?  I'm not 100% that it will fix it though